src/Symfony/Component/Console/composer.json

```diff
@@ -20,12 +20,15 @@
         "symfony/polyfill-mbstring": "~1.0"
     },
     "require-dev": {
+        "symfony/http-kernel": "~2.8|~3.0",
         "symfony/event-dispatcher": "~2.8|~3.0",
+        "symfony/dependency-injection": "~2.8|~3.0",
         "symfony/filesystem": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
         "psr/log": "~1.0"
     },
     "suggest": {
+        "symfony/dependency-injection": "",
```

Comment:
I would remove this suggestion. The DI component does not allow to add more features to the Console component.

The compiler pass is useful only for people already using the component. So the suggestion is just useless noise.